cider-velox/test/DataConvertorTest.cpp

```diff
@@ -465,6 +520,34 @@ void testToVeloxDirect<Timestamp>(CiderBatch& input,
   }
 }
 
+template <>
+void testToVeloxDirect<Date>(CiderBatch& input,
+                             const CiderTableSchema& schema,
+                             memory::MemoryPool* pool) {
+  static const int64_t SecondsInOneDay = 24 * 60 * 60;
+  std::shared_ptr<DataConvertor> convertor = DataConvertor::create(CONVERT_TYPE::DIRECT);
+  RowVectorPtr rvp = convertor->convertToRowVector(input, schema, pool);
+
+  RowVector* row = rvp.get();
+  auto* rowVector = row->as<RowVector>();
+  EXPECT_EQ(1, rowVector->childrenSize());
+  VectorPtr& child_0 = rowVector->childAt(0);
+  EXPECT_TRUE(child_0->mayHaveNulls());
+
+  auto childVal_0 = child_0->asFlatVector<Date>();
+  auto nulls_0 = child_0->rawNulls();
+  const int64_t* col_0 = reinterpret_cast<const int64_t*>(input.column(0));
+  int num_rows = input.row_num();
+
+  for (auto idx = 0; idx < num_rows; idx++) {
+    if (col_0[idx] == plugin::inline_int_null_value<int64_t>()) {
+      EXPECT_TRUE(bits::isBitNull(nulls_0, idx));
+    } else {
+      EXPECT_EQ(childVal_0->valueAt(idx), Date(col_0[idx] / SecondsInOneDay));
+    }
```

Comment:
need to add some date functionality integration tests to verify?

Reply:
This test just make sure the converter is right?
Functionality should not be tested here？(The isolated attribute of UT)
And integration tests about date are included by e2e test.